examples/dreamcast/basic/stackprotector/Makefile

```diff
@@ -4,6 +4,9 @@ KOS_CFLAGS += -fstack-protector-all
 
 GCC_MAJOR = $(basename $(basename $(KOS_GCCVER)))
 
+clean:
```

Comment:
It'd probably make more sense to revert this and add clean to the `all dist:` line inside the 4.x block.